tests/unit/containers/my-cooperations/cooperation-details/CooperationDetails.spec.jsx

```diff
@@ -59,6 +59,16 @@ const cooperationMock = {
   updatedAt: '2024-01-12T11:28:34.397Z'
 }
 
+vi.mock(
+  '~/containers/my-cooperations/cooperation-notes/CooperationNotes',
+  () => ({
+    __esModule: true,
```

Comment:
Please note that we don't need to use `__esModule: true` in the current Vitest version.

Reply:
Thank you! I've removed it